src/EnergyPlus/WaterCoils.cc

```diff
@@ -2374,28 +2372,29 @@ void SizeWaterCoil(EnergyPlusData &state, int const CoilNum)
                 CompType = cAllCoilTypes(Coil_CoolingWater); // Coil:Cooling:Water
             }
 
-            bPRINT = false; // do not print this sizing request since the autosized value is needed and this input may not be autosized (we should
-                            // print this!)
+            bool bPRINT = false; // do not print this sizing request since the autosized value is needed and this input may not be autosized (we
+                                 // should print this!)
             if (state.dataWaterCoils->WaterCoil(CoilNum).DesAirVolFlowRate == state.dataSize->DataFlowUsedForSizing) {
                 TempSize = state.dataWaterCoils->WaterCoil(CoilNum).DesAirVolFlowRate; // represents parent object has hard-sized airflow
             } else {
                 TempSize = AutoSize; // get the autosized air volume flow rate for use in other calculations
             }
 
-            bool errorsFound = false;
+            ErrorsFound = false;
             CoolingAirFlowSizer sizingCoolingAirFlow;
-            CompName = state.dataWaterCoils->WaterCoil(CoilNum).Name;
+            std::string CompName = state.dataWaterCoils->WaterCoil(CoilNum).Name;
```

Comment:
This can be a `std::string const &` (or `std::string_view`) unless you really need a local copy.